W-13547081 edit for stricter standards bsj (#61)
mulesoft/docs-api-governance-documentation: modules/ROOT/pages/create-profiles.adoc

= Applying Governance Rulesets to Specified APIs

After you identify the APIs in Exchange that 

> you want to govern, you are ready to create governance profiles. Governance profiles enable you to filter the specific sets of APIs to govern for each set of rulesets.

Reviewer: ```suggestion
you want to govern, create your governance profiles. Governance profiles enable you to filter the specific sets of APIs to govern for each set of rulesets.
```

Author: I made a slightly different change, since "you can" implies it is a requirement to identify them first, whereas it is reallyl a best practice.
